package.json

```diff
@@ -42,49 +42,49 @@
   "homepage": "https://github.com/open-formulieren/formio-renderer#readme",
   "peerDependencies": {
     "formiojs": "~4.13.0 || ~4.14.0",
-    "react": "^17.0.2"
+    "react": "^18.2.0"
   },
   "devDependencies": {
```

Comment:
feel free to make the dev tools setup the same as the formio builder. I'd rather see ESLint added to both repositories at the same time instead of this mixed set up now.

This repo was started from a very outdated boilerplate, while the formio-builder one was crafted by hand. You can be bold ;)